src/core/adt/derive/setoid.js

```diff
@@ -0,0 +1,24 @@
+const assertType = require('../../../helpers/assertType');
```

Comment:
You can use `require('folktale/helpers/assertType')` here. `folktale/` in require resolves to the root of the project.